### PR TITLE
feat: html to jsx converter added

### PR DESCRIPTION
**Pull request by diganta413**
#26 Html to jsx converter added

### CODE REVIEW

**Comment by diganta413** [comment]
Hey @AnishDe12020 is there anything wrong with my pr??

**Comment by AnishDe12020** [comment]
> Hey @AnishDe12020 is there anything wrong with my pr??

You seem to have used NPM though this project actually uses yarn. Can you remove the `package-lock.json` file, and install all dependencies with yarn so that it updates the `yarn.lock` file and push again?

**Comment by diganta413** [comment]
Hey @AnishDe12020 I have made the required changes...

File: data/tools/converters/HtmlToJsx.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { Tool } from "@/data/types";
+
+const HtmlToJsx: Tool = {
+  id: 2,
+  name: "Html To Jsx",
+  description: "Convers Html To Jsx and vice versa",
+  componentFileName: "HtmlToJsx",
+  slug: "html-to-jsx",
+  categorySlug: "converters",
```
Comment (AnishDe12020) [inline data/tools/converters/HtmlToJsx.ts:9]: I was actually working on a new feature. That requires you to add an `acceptedDataTypes` field which is an array of accepted data types. I have made it optional now but I guess you still have to add it. Just add the following - 

```ts
const HtmlToJsx: Tool = {
  id: 2,
  name: "Html To Jsx",
  description: "Convers Html To Jsx and vice versa",
  componentFileName: "HtmlToJsx",
  slug: "html-to-jsx",
  categorySlug: "converters",
  acceptedDataTypes: ["html", "jsx"],  // add this line
}

**Review by AnishDe12020 — CHANGES_REQUESTED** [review]

File: data/tools/converters/HtmlToJsx.ts (hunk 1 above)
Comment (diganta413) [reply data/tools/converters/HtmlToJsx.ts:9]: Ok

**Comment by AnishDe12020** [comment]
@all-contributors add @diganta413 for tool

**Pull request merged**